### PR TITLE
Pre eval setup

### PR DESCRIPTION
**Pull request by madison-codes**
This pull request lays the groundwork for the rest of this project eval. This PR adds code that pulls data from the endpoint provided, maps through the contents and displays it in the appropriate hierarchy. 

The final product: 
screen shot 2016-11-09 at 7 18 47 am

### CODE REVIEW

File: quizzer/src/Answer.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+export default class Answer extends React.Component {
+  render() {
+    let {title, score} = {...this.props.data};
+    return (
+      <label >
+        <input
+          className='answer'
+          type='checkbox'
```
Comment (brittanydionigi) [inline quizzer/src/Answer.js]: We likely only want to allow users to select a single answer per question. The radio element can take a `name` attribute that will limit our selection. See example [here](http://www.w3schools.com/html/tryit.asp?filename=tryhtml_input_radio).

The name should be unique per question. i.e. all of the answer elements in question one might have a `name` attribute equal to `answer-group-1`.

**Review by brittanydionigi — CHANGES_REQUESTED** [review]
Small change then this is good to go

**Comment by madison-codes** [comment]
Just pushed a commit that now allows the user to only select a single answer to each question. Did this by passing down the id of the question to each possible answer as a name. Since the names are all the same, it changes the option to only allow you to change one. 

http://g.recordit.co/83E8UXM4xA.gif

**Comment by madison-codes** [comment]
@brittanystoroz should I merge this now that the changes are made?

**Pull request merged**